content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,30 +226,75 @@ clusters, zones, or regions.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+### Availability vs Config Isolation
+
+The following list ranks control plane deployment examples by availability and configuration isolation:
+
+- One control plane per region or a single global control plane (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority. You can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
+If you use a single shared trust domain, a service account in a given namespace is globally unique - i.e. if
+two clusters `cluster-1` and `cluster-2` have the same service account `foo` in namespace `ns`, they will be
+treated the same for purposes of security. With a trust domain for each cluster, there are no
+restrictions on the uniqueness of service accounts, and namespaces. You can apply enforcement
+policies that allow or deny all traffic from one cluster (irrespective of the namespace/service) to another.
 
-- Configuration isolation: You can make configuration changes in one cluster,
-  zone, or region without impacting others.
+### Multiple meshes
+
+Multiple mesh deployments result from {{< gloss >}}mesh federation{{< /gloss >}}, with the following capabilities:
+
+- Organizational boundaries: lines of business.
+- Potential for cross vendor meshes to communicate with each
+  other if the trust bundles are setup properly.
 
-You can improve control plane availability through fail over. When a control
-plane instance becomes unavailable, workload instances can connect to
-another available control plane instance. Fail over can happen across clusters,
-zones, or regions.
+You can enable inter-mesh communication with {{< gloss >}}mesh federation{{</gloss >}}.
+When federating, each mesh can expose a set of services and identities, which all participating meshes can recognize.
 
 {{< image width="50%"
-    link="failover.svg"
-    alt="A service mesh after a control plane instance fails"
-    title="Control plane fail over"
-    caption="A service mesh after a control plane instance fails"
+    link="multi-mesh.svg"
```

Comment:
Confused about this diagram, how is this different than single mesh multiple trust domains mentioned earlier where it also says `You can apply enforcement
policies that allow or deny all traffic from one cluster (irrespective of the namespace/service) to another`

Reply:
I realize you may not have changed diagrams... you reshaped what is single mesh that could include multiple trust domains, which made me think this diagram isn't only applied to multi-meshes based on the latest doc thus the confusion.